starboard/loader_app/slot_management.cc

```diff
@@ -14,8 +14,14 @@
 
 #include "starboard/loader_app/slot_management.h"
 
+#include <stdio.h>
 #include <sys/stat.h>
 
+#include <algorithm>
+#include <fstream>
```

Comment:
See my response in the chat, but avoid using `<fstream>`, this will lead to leaks. In here, you can probably use `starboard/common/file` to read in file content, and then pass it on to the JSON parser with a string in-memory.

Reply:
Thanks, yep, I used ScopedFile from `starboard/common/file` now.